README.md

```diff
@@ -28,7 +28,7 @@ This repository contains the **source code** of the official App registry for th
     **Note**: The fields `title` and `description` are mandatory.
 
     **Note**: If you used the
-    [AiiDA Lab App cookie cutter](https://github.com/aiidalab/aiidalab-app-cutter)
+    [AiiDA lab App cookie cutter](https://github.com/aiidalab/aiidalab-app-cutter)
```

Comment:
also here we should be consistent
(please update the other instances of inconsistencies)